### ISSUE TITLE
self.training_output does not have model path

### ISSUE THREAD

**Issue by shourja123** [tim-smart/label-studio-spacy#1]
On completing a training cycle, the model gets saved to the "latest-model" dir. However, on opening a prediction , the model fails to predict since model loading does not work. Below are the logs along with the output of `print(self.train_output)`:
```
[2023-01-31 18:41:01,572] [INFO] [werkzeug::_log::225] 127.0.0.1 - - [31/Jan/2023 18:41:01] "POST /predict HTTP/1.1" 200 -
[2023-01-31 18:41:01,575] [INFO] [werkzeug::_log::225] 127.0.0.1 - - [31/Jan/2023 18:41:01] "GET /health HTTP/1.1" 200 -
[2023-01-31 18:41:01,578] [INFO] [spacy_ner::__init__::55] MODEL CHECKPOINT: fallback
[2023-01-31 18:41:01,578] [INFO] [werkzeug::_log::225] 127.0.0.1 - - [31/Jan/2023 18:41:01] "POST /setup HTTP/1.1" 200 -
[2023-01-31 18:41:01,586] [ERROR] [spacy_ner::predict::103] model has not been trained yet
{'job_id': 1675162945}
```
As you can see the value of self.train_output only contains the job_id. Any idea how to approach this?
